test/functional/test-amp-context.js

```diff
@@ -61,6 +59,24 @@ describe('3p ampcontext.js', () => {
     windowMessageHandler = undefined;
   });
 
+  it('should send message when report3pError_()', () => {
+    win.name = generateSerializedAttributes();
+    const context = new AmpContext(win);
+    expect(context).to.be.ok;
+    context.report3pError_();
```

Comment:
Same question again, the constructor already called `report3pError_()` why we need to call it here?

Reply:
i needed it to test addEventListener call before, but forgot to remove. Will change now.